core/app/[locale]/layout.tsx

```diff
@@ -37,13 +42,17 @@ export async function generateMetadata(): Promise<Metadata> {
     fetchOptions: { next: { revalidate } },
   });
 
-  const title = data.site.settings?.storeName ?? '';
+  const storeName = data.site.settings?.storeName ?? '';
+
+  const { pageTitle, metaDescription, metaKeywords } = data.site.settings?.seo || {};
 
   return {
     title: {
-      template: `${title} - %s`,
-      default: title,
+      template: `%s - ${storeName}`,
```

Comment:
intentional stylistic change here - page first, store name second (same thing we do on stencil)